backends/tc/backend.cpp

```diff
@@ -354,29 +354,52 @@ void ConvertToBackendIR::updateDefaultMissAction(const IR::P4Table *t, IR::TCTab
                 if (defaultActionProperty->isConstant) {
                     tabledef->setDefaultMissConst(true);
                 }
+                auto annoList = defaultActionProperty->getAnnotations()->annotations;
+                bool isTCMayOverride = false;
```

Comment:
getSingle function shall be used instead of the loop.
overrideAnno = defaultActionProperty->getAnnotations()->getSingle(ParseTCAnnoations::tcMayOverride)

Reply:
Updated the code